src/Uno.UI/UI/Xaml/FrameworkElement.crossruntime.cs

```diff
@@ -240,6 +239,10 @@ void InvokeUnloadedWithTry()
 
 		private protected virtual void OnUnloaded() { }
 
+#if DEBUG && UNO_REFERENCE_API
+		private void OnGenericPropertyUpdated(DependencyPropertyChangedEventArgs args) { }
+#endif
```

Comment:
This is already defined for Debug in `.skia.cs` partial, so it will fail to build

Reply:
should i be using `__NETSTD_REFERENCE__` instead?
i added this line, because when building Uno.UI with Uno.UI-Reference-only.slnf & net8.0, 
the compiler were complaining about no such definition

Reply:
In that case, yes, I think `__NETSTD_REFERENCE__` will work.

Currently, `UNO_REFERENCE_API` is incorrectly defined for Reference, Skia, and Wasm. It should have been Reference only.